lib/vagrant-service-manager/services/openshift.rb

```diff
@@ -0,0 +1,24 @@
+module Vagrant
+  module ServiceManager
```

Comment:
If this service requires docker, shouldn't we somehow have it call docker in case the user didn't already specify it?  It should also detect if docker was listed and not re-call it.